doc/index.md

```diff
@@ -345,7 +348,7 @@ Flushes all buffered data on the socket. Returns true on success. Returns false
 
 ### `h1_connection:read_request_line(timeout)` <!-- --> {#http.h1_connection:read_request_line}
 
-Gets the request method, requested path and HTTP version for an incoming request. On error, returns nil, an error message and an error number.
+Returns information about in incoming request. Returns the `request method`, `requested path` and `HTTP version` for an incoming request. On error, returns nil, an error message and an error number.
```

Comment:
Importantly this method *reads* from the socket (it's not information already cached)